db/migrate/20181018233104_add_user_id_to_posts.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserIdToPosts < ActiveRecord::Migration[5.2]
+  def change
+    add_column :posts, :user_id, :integer
```

Comment:
The best solution here is explicitly adding the relation here and in your models users and posts.